src/components/textarea/bl-textarea.css

```diff
@@ -0,0 +1,137 @@
+:host {
+  display: inline-block;
+  width: 200px;
+  position: relative;
+}
+
+textarea {
+  outline: none;
+  box-sizing: border-box;
+  border: solid 1px var(--bl-color-border);
+  width: 100%;
+  font: var(--bl-font-title-3-regular);
+  padding:var(--bl-size-2xs) var(--bl-size-xs);
+  margin: 0;
+  border-radius: 4px;
+  color: var(--bl-color-content-primary);
+}
+
+textarea:disabled{
+  background-color: var(--bl-color-secondary-background);
```

Comment:
@muratcorlu We applied the same style in the input component. Secondary-background for background and content-passive for text in disabled versions.

Reply:
But you know, if we use blue for the secondary color, then disabled inputs will be also blue, right? With our color palette issue, we'll already try to address those issues but since this is a new component, maybe we can start to avoid using secondary colors instead of content colors.

Reply:
So what is our decision about that?

Reply:
we can continue with the colors in the design @ozkersemih